packages/notifications/push/amplify_push_notifications_pinpoint/lib/src/pinpoint_provider.dart

```diff
@@ -38,6 +39,19 @@ class PinpointProvider implements ServiceProviderClient {
       'pinpoint.campaign.treatment_id';
   bool _isInitialized = false;
 
+  static final _userAgent =
+      'amplify-flutter/${packageVersion.split('+').first}/pushnotifications';
```

Comment:
Spoke with Abdallah and we're just gonna add `pushnotifications` instead of trying to replace the `amplify-flutter` component.

```suggestion
      'pushnotifications';
```

Reply:
Can revert the version dependency too if you want. The minor version will always be the same as `amplify_flutter` so we don't need to include the push version.